CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+UNRELEASED
+======
+
+* Added support for passing multiple separated values with one param. Added `paramSeparator` to `@Spec` interface, which allows for specifying regex which is used to the delimitation of param value. For example controler method:
```

Comment:
Added support for passing multiple values as a single HTTP parameter. Then new `paramSeparator` attribute of `@Spec` can be used to define the separator (e.g. comma). For example the following controller method:

Reply:
Additionally:
- let's start with simple char/string separators, without regex support -- regexp might become dangerous in some cases, and I think that comma will suffice for most use-cases